soft4pes/util/conversions.py

```diff
@@ -0,0 +1,60 @@
+"""
+Reference-frame conversions.
```

Comment:
It should also be moved in a different folder as it relates to all systems

Reply:
Do you mean that the module docstring should highlight these keywords and how they are related to different transformations, or that they should appear in the function docstring?

Reply:
> It should also be moved in a different folder as it relates to all systems

The conversions are used by both controllers and models. Therefore I think the current path is all right. The folder could be renamed from `util ` to `com`  or `common`

Reply:
I mean that this file is not complete as it misses other transformations.

I noticed that it is already in a different folder. I would name this folder as auxFunctions or auxFiles or something like this as it will have auxiliary functions that are used by different files

Reply:
I will make a note about adding the conversions.

For the path, it seems (based on a quick search) that using `utils` is the one that is usually used. I shall rename it as that